hack/arktos-up-scale-out-poc.sh

```diff
@@ -681,6 +683,49 @@ if [ "${IS_RESOURCE_PARTITION}" == "true" ]; then
 fi
 
 if [ "${IS_RESOURCE_PARTITION}" != "true" ]; then
+  # Applying mizar cni
+  if [[ "${CNIPLUGIN}" == "mizar" ]]; then
+    ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create configmap system-source --namespace=kube-system --from-literal=name=arktos --from-literal=company=futurewei
+
+    # Creating mizar crds
+    echo "Creating mizar crds ......."
+    cp "${KUBE_ROOT}/third_party/mizar/mizar-crds.yaml" mizar-crds.yaml
+    ${KUBECTL} apply -f mizar-crds.yaml
+    rm mizar-crds.yaml
+
+    if [[ "${IS_SECONDARY_TP}" == "false" ]]; then
+      # Deploying mizar daemonset
+      echo "Deploying mizar daemonset ......."
+      cp "${KUBE_ROOT}/third_party/mizar/mizar-daemon.yaml" mizar-daemon.yaml
+      sed -i -e "s@{{network_provider_version}}@${MIZAR_VERSION}@g" mizar-daemon.yaml
+      ${KUBECTL} apply -f mizar-daemon.yaml
+      rm mizar-daemon.yaml
+    fi
+
+    # Deploying mizar daemon to tp master
+    echo "Deploying mizar daemon to TP master ......."
+    cp "${KUBE_ROOT}/third_party/mizar/mizar-daemon-tpmaster.yaml" mizar-daemon-tpmaster.yaml
+    sed -i -e "s@{{network_provider_version}}@${MIZAR_VERSION}@g" mizar-daemon-tpmaster.yaml
+    sed -i -e "s@{{tp_master_name}}@${API_HOST}@g" mizar-daemon-tpmaster.yaml
+    ${KUBECTL} apply -f mizar-daemon-tpmaster.yaml
+    rm mizar-daemon-tpmaster.yaml
+
+    # Place mizar operator
```

Comment:
mizar. operator is on each TP only. so. this should have a flat to not deploy it on RPs.

Reply:
The outter if statement is for TP only (line 685)